lib/embed/embed.dart

```diff
@@ -27,7 +27,7 @@ import '../polymer/iron.dart';
 import '../polymer/paper.dart';
 import '../services/_dartpadsupportservices.dart';
 import '../services/common.dart';
-import '../services/dartservices.dart';
+import '../services/dartservices.dart' hide Position;
```

Comment:
I'm curious why Position needs to be hidden. Is it deprecated, or clashes with another import?

Reply:
Clashes with a CodeMirror class of the same name.

Reply:
...aaaaand this is no longer true. Position has been refactored out of the API.